frontend/src/components/auth/components/sign-wrapper-form/sign-wrapper-form.tsx

```diff
@@ -0,0 +1,21 @@
+import style from './sign-wrapper-form.module.scss';
```

Comment:
actually, we don't need this component
with wrapper should be as `frontend/src/components/auth/auth.tsx`component